app/models/illumina_htp/plate_purposes.rb

```diff
@@ -85,21 +85,13 @@ module IlluminaHtp::PlatePurposes
     'Cherrypicked' => 'illumina_b_shared'
   }
 
-  PF_PLATE_PURPOSES_TO_REQUEST_CLASS_NAMES = [
-    ['PF Cherrypicked', 'PF Shear', :initial],
-    ['PF Shear', 'PF Post Shear'],
-    ['PF Post Shear', 'PF Post Shear XP'],
-    ['PF Post Shear XP', 'PF Lib XP'],
-    ['PF Lib XP', 'PF Lib XP2']
-  ]
-
   PLATE_PURPOSES_TO_REQUEST_CLASS_NAMES = [
```

Comment:
This is all stuff that would probably better belong in a yaml file with a corresponding factory, as its pretty much only used when seeding the databse. Part of the seeds project perhaps.